api/src/main/java/org/openmrs/module/fhir/api/util/FHIRObsUtil.java

```diff
@@ -379,6 +380,13 @@ public static Obs generateOpenMRSObs(Observation observation, List<String> error
 		}
 		return obs;
 	}
+
```

Comment:
Where is this method will use? if there is no usage the purpose will not be address